documentation/proc-pages/physics-models/plasma_geometry.md

```diff
@@ -1,105 +1,682 @@
+| `thetao`   | Half-angle of arc describing outboard surface |
+
+
+| Output Variable | Description                          |
+|----------|--------------------------------------|
+| `xsecta`       | Plasma cross-sectional area [m^2] |
+
+
+$$
+\mathtt{xsecta} = \mathtt{xo}^2 \times (
+            \mathtt{thetao} - \cos{(\mathtt{thetao})} \times \sin({\mathtt{thetao}})
+        ) + \mathtt{xi}^2 \times (\mathtt{thetai} - \cos{(\mathtt{thetai})} \times \sin{(\mathtt{thetai}))}
+$$
+
+
+## Legacy claculations
+
+### STAR Code plasma surface area | `surfa()` 
+
+This function finds the plasma surface area, using the
+revolution of two intersecting arcs around the device centreline.
+This calculation is appropriate for plasmas with a separatrix.
+It was the original method in PROCESS[^6].
+
+
+| Input Variable | Description                          |
+|----------|--------------------------------------|
+| `rminor`, $a$        | Plasma minor radius (m)              |
+| `rmajor`, $R$        | Plasma major radius (m)              |
+| `kappa`, $\kappa$      | Plasma separatrix elongation         |
+| `triang`, $\delta$      | Plasma separatrix triangularity      |
+
+| Output Variable | Description                          |
+|----------|--------------------------------------|
+| `sa`       | Plasma total surface area (m^2) |
+| `so`   | Plasma outboard surface area (m^2) |
+
+$$
+\mathtt{radco} = a \frac{(1.0 + (\kappa^2 + \delta^2 - 1.0)}{(2.0 \times (1.0 + \delta))}
+$$
+
+$$
+\mathtt{b} = \kappa \times a
+$$
+
+$$
+\mathtt{thto} = \arcsin{(\mathtt{b}/\mathtt{radco})}
+$$
+
+$$
+\mathtt{so} = 4.0\pi \times \mathtt{radco} \times ((R + a - \mathtt{radco}) \times \mathtt{thto} + \mathtt{b})
+$$
+
+Inboard side
+
+
+$$
+\mathtt{radci} = a \frac{(1.0 + (\kappa^2 + \delta^2 - 1.0)}{(2.0 \times (1.0 - \delta))}
+$$
+
+$$
+\mathtt{b} = \kappa \times a
+$$
+
+$$
+\mathtt{thti} = \arcsin{(\mathtt{b}/\mathtt{radci})}
+$$
+
+$$
+\mathtt{si} = 4.0\pi \times \mathtt{radci} \times ((R - a + \mathtt{radci}) \times \mathtt{thti} - \mathtt{b})
+$$
+
+$$
+\mathtt{sa} = \mathtt{so} + \mathtt{si}
+$$
+
+### Plasma poloidal perimeter calculation | `perim()`
+
+This function finds the plasma poloidal perimeter, using the
+revolution of two intersecting arcs around the device centreline.
+This calculation is appropriate for plasmas with a separatrix.
+
+| Input Variable | Description                          |
+|----------|--------------------------------------|
+| `rminor`, $a$        | Plasma minor radius (m)              |
+| `kappa`, $\kappa$      | Plasma separatrix elongation         |
+| `triang`, $\delta$      | Plasma separatrix triangularity      |
+
+| Output Variable | Description                          |
+|----------|--------------------------------------|
+| `perim`       | Plasma poloidal perimeter length (m) |
+
+Inboard arc
+
+$$
+\mathtt{denomi} = \frac{(\delta^2 + \kappa^2 - 1.0)}{(2.0 \times (1.0 - \delta)) + \delta}
```

Comment:
I think the `+\delta` at the end of the denominator is supposed to be outside the fraction?